### DISCUSSION TITLE
How do I acheive copying %artist% to %albumartist%

### DISCUSSION THREAD

**Discussion by bonelifer** [puddletag/puddletag#633] [Q&A] [answered]
I'd like to be able to have an action that allows me to copy the %artist% tag to %albumartist% in the ACTIONS functions. Not sure how to do that, even after looking at the action options currently available.

**Comment by audiomuze** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/6852511/107193684-c5b8e700-69f7-11eb-97a1-a08011d2efaa.png)

![image](https://user-images.githubusercontent.com/6852511/107193828-f13bd180-69f7-11eb-81e2-bf21954de4b3.png)
